azure-pipelines.yml

```diff
@@ -6,6 +6,16 @@ variables:
   - group: darwin
 
 trigger:
+  batch: true
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - report/report.tex
+    - docs/README.md
```

Comment:
What's the purpose of the `docs/README.md` exclude? 🤔

Reply:
That no build is triggered if only the report or readme changes

Reply:
Makes sense, imo.

Reply:
Yeah sure, but it's README.md instead of docs/README.md, isn't it?

Reply:
Oh, yes of course, sorry ^^

Reply:
If we decide to leave this exclusions i need to add them to the pr section aswell